testcases/crypto/ec_func.c

```diff
@@ -464,31 +464,30 @@ CK_RV run_DeriveECDHKey()
                     // key and B's public key
 
                     if (is_ep11_token(SLOT_ID)) {
-                        /* ep11token implements PKCS#11 v2.20 */
```

Comment:
Please note that the removed code mentions that the EP11 library would implement PKCS#11 v2.20. This is not true as far as I can tell. I checked the PKCS#11 v2.20 standard, and it already uses the CK_ECDH1_DERIVE_PARAMS as mechanism parameter. CKM_ECDH1_DERIVE was added with v2.11 and uses CK_ECDH1_DERIVE_PARAMS since then.

So obviously EP11 did never comply with PKCS#11 with CKM_ECDH1_DERIVE.